### ISSUE TITLE
legends for plots in README.md 

### ISSUE THREAD

**Issue by lauracion** [softloud/parameterpal#19]
Hey,

Just noticed that the normal plot legend may need to have "within 0.5 of 0.2" changed to "within 0.2 of 0.5".

Also, the beta plot legend was hard to follow for me. Maybe you could use: "Assuming a beta distribution (dotted line), with expected value 0.3 (vertical dashed line), and 0.8 of values falling within 0.2 of 0.3, we approximate the distribution beta(2.30691, 5.38279). Parameters in the title are rounded. The shaded region shows the desired interval of 0.8 of values falling within 0.2 of 0.3."

Now that I wrote that I am wondering if using 80% would not be clearer to the reader than 0.8. But that will depend a lot on your readership 😄 

Cheers!

**Comment by softloud** [comment]
I *really* appreciate you taking a look at the text, @lauracion :heart: 

You know when you rewrite something too many times in a row to know if it flows anymore? Anyways, i knew I couldn't edit it until I'd rested it; but fresh eyes can *do it now*. If you'd asked me what the number one thing to be helpful is right now, I'd have said this. :vulcan_salute: with you, grrl.

**Comment by softloud** [comment]
- [x] typo
- [x] caption
- [x] shift to % for width, I like this note

**Comment by softloud** [comment]
Updated caption, would appreciate it if you could take another look. A key piece of information I think I should convey is that it's an approximation, and that, for some parameters, _not quite_ the same percentage fall in that region; but close enough.

Haven't  found the right wording yet, but I think we're getting closer. 

``` r
library(parameterpal)

beta_plot(0.3, 0.1, 0.8)
```

![](https://i.imgur.com/zMugVau.png)

<sup>Created on 2020-08-18 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>
